docs/clusters/Wulver_filesystems.md

```diff
@@ -0,0 +1,17 @@
+# Wulver Filesystems
+
+The Wulver environment is quite a bit like Lochness, but there are some key differences, especially in filesystems and SLURM partitions and priorities.
+
+ Wulver Filesystems are deployed with more attention to PI ownership / group efforts:
+
+1. The `$HOME` directory is not intended for primary storage and will have only 50GB quota. To run the simulations, compilations, etc., users need to use a project directory which has 2TB of storage per PI group. Students can store their files under their corresponding PI’s UCID in the `/project` directory.  For example, if PI’s UCID is `doctorx`, then students need to use the `/project/doctorx/` directory. 
```

Comment:
The `$HOME` directory is not intended for primary storage and has a 50GB quota. The main location for storing files is the group project directory which has 2TB of storage per PI group.